<launch>
    <node pkg="riser_inspection" type="waypoint_mission_node" name="riser_inspection" output="screen">
        <param name="gps_topic"         type="string"   value="/dji_osdk_ros/gps_position" />
        <param name="rtk_topic"         type="string"   value="/dji_osdk_ros/rtk_position" />
        <param name="attitude_topic"    type="string"   value="/dji_osdk_ros/attitude" />
        <param name="height_topic"       type="string"   value="/dji_osdk_ros/height_above_takeoff"/>
        <param name="root_directory"    type="string"   value="/home/vant3d/Documents"/>
        <param name="wait_time"         type="int"      value="6"/>     <!-- SECONDS-->
        <param name="riser_distance"    type="int"      value="5"/>     <!--METERS-->
        <param name="riser_diameter"    type="int"      value="300"/>   <!--MILLIMETER-->
        <param name="horizontal_points" type="int"      value="5"/>     <!--Nº of horizontal stops-->
        <param name="vertical_points"   type="int"      value="4"/>     <!--Nº of vertical stops at each horizontal point-->
        <param name="delta_H"           type="int"      value="15"/>    <!--DEGREES-->
        <param name="delta_V"           type="int"      value="300"/>   <!--MILLIMETERS-->
        <param name="vertical_error"    type="double"   value="0.1"/>   <!--METERS-->
        <param name="horizontal_error"    type="double"   value="0.2"/>   <!--METERS-->
    </node>
</launch>